static/css/all.css

```diff
@@ -0,0 +1,4423 @@
+/*!
```

Comment:
@Thanson16 can this be renamed to be clear that it's font-awesome?

Reply:
fixed